scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala

```diff
@@ -211,14 +224,20 @@ class HtmlRenderer(rootPackage: Member, val members: Map[DRI, Member])(using ctx
               raw("&nbsp;Back to top")
             )
           ),
-          raw("Generated by&nbsp;"),
-          a(href := "https://github.com/lampepfl/dotty/tree/master/scaladoc")(
-            img(
-              src := resolveRoot(link.dri, "images/scaladoc_logo.svg"),
-              alt := "scaladoc",
-              cls := "scaladoc_logo"
+          div(id := "footer-socials")(
```

Comment:
maybe instead of ids we can use jus a class and then based on parent class style it in css?

so instead of `#footer-socials` we are using `.footer .socials`